src/smart-components/portfolio/portfolios.js

```diff
@@ -17,102 +15,90 @@ import { fetchPortfolios } from '../../redux/actions/portfolio-actions';
 import PortfolioCard from '../../presentational-components/portfolio/porfolio-card';
 import createPortfolioToolbarSchema from '../../toolbar/schemas/portfolios-toolbar.schema';
 import ContentGalleryEmptyState, { EmptyStatePrimaryAction } from '../../presentational-components/shared/content-gallery-empty-state';
+import asyncFormValidator from '../../utilities/async-form-validator';
+
+const debouncedFilter = asyncFormValidator((value, dispatch, filteringCallback) => {
+  filteringCallback(true);
+  dispatch(fetchPortfolios(value, defaultSettings)).then(() => filteringCallback(false));
```

Comment:
- any filtering will reset the pagination to the default options.
maybe use the pagination option here:
const debouncedFilter = asyncFormValidator((value, dispatch, filteringCallback, pagination) => {
  filteringCallback(true);
  dispatch(fetchPortfolios(value, pagination)).then(() => filteringCallback(false));
}, 1000);


and then:
-------
 const handleFilterItems = value => {
    stateDispatch({ type: 'setFilterValue', payload: value });
    debouncedFilter(value, dispatch, isFiltering => stateDispatch({ type: 'setFilteringFlag', payload: isFiltering }), meta);
  };

Reply:
@lgalis Is that the same as the bug I mentioned this morning? 
 https://projects.engineering.redhat.com/browse/SSP-630

Reply:
@lgalis we can only send limit not offset. Offset has to be reset but you are right that limit can be kept.

Reply:
> @lgalis Is that the same as the bug I mentioned this morning?
> https://projects.engineering.redhat.com/browse/SSP-630

It is the same.

Reply:
@Hyperkid123 - sounds good.